### ISSUE TITLE
Make the alpha pagination (# of people that start with __) a semantic <nav>

### ISSUE THREAD

**Issue by erinchambers** [CDRH/earlywashingtondc#262]
Follow-up task to #258 - this should be wrapped by a `<nav>` element with a unique `aria-label=""` to distinguish it from other menus on the page (main navigation, numerical pagination, etc.).

For the ARIA label, I'd suggest `aria-label="Browse by Letter"`
